src/Bridges/Constraint/bridges/norm_to_power.jl

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    NormToPowerBridge{T,F} <: Bridges.Constraint.AbstractBridge
+
+`NormToPowerBridge` implements the following reformulation:
+
+  * ``(t, x...) \\in NormCone(p, 1+d)`` into ``(r_i, t, x_i) \\in PowerCone(1 / p)``
+    for all ``i``, and ``\\sum\\limits_i r_i == t``.
+
+## Source node
+
+`NormToPowerBridge` supports:
+
+  * `F` in [`MOI.NormCone`](@ref)
+
+## Target nodes
+
+`NormToPowerBridge` creates:
+
+  * `F` in [`MOI.PowerCone{T}`](@ref)
```

Comment:
Maybe we can cite the same book as https://github.com/jump-dev/Convex.jl/blob/c6a21df2979dc1eaf60674f7f4b1324a4f30f8b8/src/atoms/second_order_cone/rationalnorm.jl#L1-L130

Reply:
Yes, I guess I need to add a rational norm bridge. But the implementation is always a little messy.